plugins/security/nexus-ldap-plugin/src/main/java/org/sonatype/security/realms/ldap/internal/SupportBundleCustomizerImpl.groovy

```diff
@@ -1,90 +0,0 @@
-/*
- * Sonatype Nexus (TM) Open Source Version
- * Copyright (c) 2008-2015 Sonatype, Inc.
- * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
- *
- * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
- * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
- *
- * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
- * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
- * Eclipse Foundation. All other trademarks are the property of their respective owners.
- */
-package org.sonatype.security.realms.ldap.internal
-
-import com.sonatype.security.ldap.realms.persist.model.io.xpp3.LdapConfigurationXpp3Reader
-import com.sonatype.security.ldap.realms.persist.model.io.xpp3.LdapConfigurationXpp3Writer
-import org.sonatype.nexus.configuration.application.ApplicationDirectories
-import org.sonatype.nexus.supportzip.GeneratedContentSourceSupport
-import org.sonatype.nexus.supportzip.SupportBundle
-import org.sonatype.nexus.supportzip.SupportBundleCustomizer
-import org.sonatype.sisu.goodies.common.ComponentSupport
-
-import javax.inject.Inject
-import javax.inject.Named
-import javax.inject.Singleton
-
-import static com.google.common.base.Preconditions.checkNotNull
-import static org.sonatype.nexus.supportzip.SupportBundle.ContentSource.Priority.HIGH
-import static org.sonatype.nexus.supportzip.SupportBundle.ContentSource.Type.SECURITY
-
-/**
- * LDAP {@link SupportBundleCustomizer}.
- *
- * @since 2.7
- */
-@Named
-@Singleton
-public class SupportBundleCustomizerImpl
-    extends ComponentSupport
-    implements SupportBundleCustomizer
```

Comment:
As LDAP configuration is plain simple (one table) and have very small cardinality (ie. one would have 4-5 servers configured max? Hence, 4-5 records in table), it uses the shared "config" database managed by `org.sonatype.nexus.internal.orient.ConfigDatabase` meant for shared config. And that class does already declare SupportBundleCustomizer and include the DB in support bundle.